src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,315 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+    private static final String ADDITION_MESSAGE = "Addition test failed";
+    private static final String SUBTRACTION_MESSAGE = "Subtraction test failed";
+    private static final String DIVISION_MESSAGE = "Division test failed";
+    private static final String MULTIPLICATION_MESSAGE = "Multiplication test failed";
+    private static final String RAISING_TO_A_POWER_MESSAGE = "Raising to a power test failed";
+    private static final double DELTA = 0.0001;
+    private static Calculator calculator;
+    private double actual;
+    private double expected;
+
+    @BeforeAll
+    static void beforeAll() {
+        calculator = new Calculator();
+    }
+    // ------------Addition------------
+
+    @Test
+    void additionWithTwoPositiveValues_Ok() throws ValidationException {
```

Comment:
перейменуй всі назви методів [README](https://github.com/mate-academy/jv-lambda-calc/blob/master/README.md)
```suggestion
    void calculate_additionWithTwoPositiveValues_Ok() throws ValidationException {
```